src/components/atoms/Button/Button.tsx

```diff
@@ -99,33 +101,34 @@ const SButton = styled.button<IButtonProps>`
   display: inline-flex;
   justify-content: center;
   align-items: center;
-  width: ${({ fullWidth }) => fullWidth && '100%'};
-  padding: ${({ sizing }) => paddings[sizing]};
+  width: ${({ fullWidth = false }) => fullWidth && '100%'};
+  padding: ${({ sizing = 'default' }) => paddings[sizing]};
   font-family: ${typography.primary};
-  font-size: ${({ sizing }) => fontSizes[sizing]};
+  font-size: ${({ sizing = 'default' }) => fontSizes[sizing]};
   line-height: 1.2;
   font-weight: ${typography.weights.semibold};
   cursor: pointer;
-  border: 1px solid ${({ styling }) => backgroundColors[styling]};
+  border: 1px solid ${({ styling = 'primary' }) => backgroundColors[styling]};
   border-radius: 8px;
-  color: ${({ styling, contrastColor }) => (contrastColor ? contrastColor : fontColors[styling])};
-  background-color: ${({ styling }) => backgroundColors[styling]};
+  color: ${({ contrastColor }) => contrastColor};
+  color: ${({ styling = 'primary', contrastColor }) => (contrastColor ? contrastColor : fontColors[styling])};
+  background-color: ${({ styling = 'primary' }) => backgroundColors[styling]};
   transition: all 140ms ease-in-out;
 
   &:not(:disabled) {
-    box-shadow: ${({ styling }) => boxShadows[styling] || 'none'};
+    box-shadow: ${({ styling = 'primary' }) => boxShadows[styling] || 'none'};
   }
 
   &:active:not(:disabled),
   &:focus:not(:disabled) {
-    box-shadow: ${({ styling }) => activeBoxShadows[styling]};
+    box-shadow: ${({ styling = 'primary' }) => activeBoxShadows[styling]};
     ${({ styling }) => (styling === 'contrastLink' || styling === 'link' ? null : 'outline: none')};
   }
 
   &:hover:not(:disabled) {
     opacity: 0.8;
-    color: ${({ styling }) => hoverFontColors[styling]};
-    background-color: ${({ styling }) => activeBackgroundColors[styling]};
+    color: ${({ styling = 'primary' }) => hoverFontColors[styling]};
+    background-color: ${({ styling = 'primary' }) => activeBackgroundColors[styling]};
```

Comment:
I think it will look cleaner to have
```ts
SButton.defaultProps = {
  styling: 'primary',
  sizing: 'default',
}
```
So we don't have to repeat that login in the props func all the time

Reply:
Good point, I think `defaultProps` were previously declared on the wrapper and not on the _styled component_, hence TS wasn't able to infer them. I'll update it 👌